README.md

```diff
@@ -18,3 +18,4 @@ After downloading the required files and having started the AI-Lab you can conne
 ## Additional Links
 
 * [Developer Guide](https://github.com/exasol/ai-lab/blob/2.0.0/doc/developer_guide/developer_guide.md)
+* [Troubleshooting](doc/user_guide/troubleshooting.md)
```

Comment:
I would move it above the developer guide, because Troubleshooting is more important for the user then the developer guide